### PR TITLE
Test Listings API Frontend

### PR DESCRIPTION
**Pull request by meetkumar-tatva**

### CODE REVIEW

File: src/app/modules/test/services/test.service.ts [hunk 1]
```diff
@@ -68,4 +69,57 @@ export class TestService {
...
+  getTests(
+    currentPageIndex: number,
+    pageSize: number,
+    searchQuery: string | null,
+    groupId: number | null,
+    status: number | null,
+    date: Date | null,
+    sortField: string | null,
+    sortOrder: string | null
+  ) {
```
Comment (Trusha-Tatvasoft) [inline src/app/modules/test/services/test.service.ts:94]: create interface for parms

Comment (meetkumar-tatva) [reply src/app/modules/test/services/test.service.ts]: Done

File: src/app/modules/test/containers/test/test.component.ts [hunk 2]
```diff
@@ -1,60 +1,244 @@
...
+      currentPageIndex: this.currentPageIndex,
+      pageSize: this.pageSize,
+      searchQuery: searchQuery,
+      groupId: groupId,
+      status: status,
+      date: filterDate,
+      sortField: this.sortField,
+      sortOrder: this.sortOrder,
+    };
+    this.testService.getTests(data).subscribe((res: any) => {
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/modules/test/containers/test/test.component.ts:123]: don't use type any

Comment (meetkumar-tatva) [reply src/app/modules/test/containers/test/test.component.ts]: Resolved

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

**Review by Trusha-Tatvasoft — APPROVED** [review]

**Pull request merged**